Sources/swift-build/--show-dependencies.swift

```diff
@@ -90,3 +92,26 @@ private final class DotDumper: DependenciesDumper {
         }
     }
 }
+
+private final class JsonDumper: DependenciesDumper {
+    func dump(dependenciesOf rootpkg: Package) {
+
+        func recursiveWalk(rootpkg: Package, isLast: Bool = true) {
+            print("{")
+            print("\"name\":\"\(rootpkg.name)\",")
+            print("\"url\":\"\(rootpkg.url)\",")
+            let version = rootpkg.version?.description ?? "unspecified"
+            print("\"version\":\"\(version)\",")
+            print("\"path\":\"\(rootpkg.path)\",")
+            print("\"dependencies\": [")
```

Comment:
You should use OutputByteStream for this... it will be more efficient, will properly quote things, and will be more readable (IMHO, at least).